src/views/Gallery.vue

```diff
@@ -180,10 +180,11 @@ export default {
 </script>
 
 <style lang="sass" scoped>
-.nest-heading
-  font-size: .9em
-  font-weight: 400
-  text-transform: uppercase
+
```

Comment:
This was also caused by an accidental re-indent. @t8 be careful with indents, if you indent one thing in the beginning, it might just nest everything below it.

Reply:
This is why it's more clear to leave one tab in the style tag (and also the script), for me at least
![image](https://user-images.githubusercontent.com/30638105/87634510-ae559c00-c73d-11ea-9f98-03882908ba22.png)
![image](https://user-images.githubusercontent.com/30638105/87634537-c2010280-c73d-11ea-9104-ebf54537dd4a.png)

Reply:
We should talk about this, to make something a standard. In my opinion, since everything inside a tag is it's children, everything should we indeted with one tab (script, and style tags' content)

Reply:
I think I agree that everything should have at least one indent, though I'm not sure how to automate the checking for this process.